code/modules/urist/mob/spoopymobs.dm

```diff
@@ -36,24 +156,15 @@
 
 /mob/living/simple_animal/hostile/urist/zombie/death()
 	. = ..()
-	if(regen)
-		var/tempnameholder = src.name
-		var/tempdescholder = src.desc
-		src.name = "corpse"
-		src.desc = "It's a corpse. Very dead."
 
-		var/matrix/M = matrix() //shamelessly stolen from human update_icons
-		M.Turn(90)
-		M.Translate(1,-6)
-		src.transform = M
-
-		src << "You begin to regenerate. This will take about [regen_delay/600] minutes."
+	if(regen)
+		to_chat(src, "You begin to regenerate. This will take about [regen_delay/600] minutes.")
 		spawn(regen_delay)
 			var/matrix/N = matrix()
+			N.Turn(270)
```

Comment:
Gah. It used to work fine originally, but it got broken during the last merge. I changed it to 270 because before it was the opposite - they would be alive, but display lying down.

Reply:
_Burned to the goddamn ground, **finally**._